.github/workflows/transition.yml

```diff
@@ -12,11 +12,32 @@ on:
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
```

Comment:
I think we should create a separate workflow file for the playwright test. This add a lot of things to the main file which already contains several tests. it makes it unclear what is essential for which part of the test.